pool/src/auctions/backstop_interest_auction.rs

```diff
@@ -20,10 +24,8 @@ pub fn create_interest_auction_data(e: &Env, backstop: &Address) -> AuctionData
         block: e.ledger().sequence() + 1,
     };
 
-    let reserve_list = storage::get_res_list(e);
     let mut interest_value = 0; // expressed in the oracle's decimals
-    for i in 0..reserve_list.len() {
-        let res_asset_address = reserve_list.get_unchecked(i);
+    for res_asset_address in assets.iter() {
         // don't store updated reserve data back to ledger. This will occur on the the auction's fill.
         let reserve = pool.load_reserve(e, &res_asset_address, false);
```

Comment:
Is there a test that verifies invalid reserves throws an error?

Reply:
I didn't notice in the test diff.

- [ ] Add negative test for set of addresses with an invalid reserve address